### ISSUE TITLE
Use of deprecated '.Site.DisqusShortname'

### ISSUE THREAD

**Issue by vanillaiice** [Junyi-99/hugo-theme-anubis2#61]
Hi, using `hugo v0.133.1+extended` gives the following error, preventing the site from running:

```sh
Start building sites … 
hugo v0.133.1+extended linux/amd64 BuildDate=unknown

WARN  deprecated: .Site.Author was deprecated in Hugo v0.124.0 and will be removed in a future release. Use taxonomies instead.
ERROR deprecated: .Site.DisqusShortname was deprecated in Hugo v0.120.0 and will be removed in Hugo 0.134.0. Use .Site.Config.Services.Disqus.Shortname instead.
Built in 229 ms
Error: error building site: logged 1 error(s)

```

**Comment by Junyi-99** [comment]
Hi, thanks for reporting this. I will fix this this weekend

**Comment by Junyi-99** [comment]
@vanillaiice fixed by #62
